Makefile

```diff
@@ -21,6 +21,10 @@ deps:
 	@go mod tidy -v
 	@go mod vendor
 
+# Run linter
+lint:
```

Comment:
can we have docker'ed version of that target? `golangci-lint` may vary from dev to dev

Reply:
Hmm. Is it really useful and used by anyone? CI never does it and I can't believe any developer will run it this way locally, it just takes too much time.

Reply:
TBH, when i have any problems with the linter in CI, i run docker'ed version just to be sure that it can be reproduced locally (and local output is more convenient to look at when fixing issues). but agree that it is totally useless unless we fix version in CI